To: vim_dev@googlegroups.com
Subject: Patch 7.3.163
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.163
Problem:    For the default of 'shellpipe' "mksh" and "pdksh" are not
	    recognized.
Solution:   Recognize these shell names.
Files:	    src/option.c


*** ../vim-7.3.162/src/option.c	2011-03-22 14:35:01.000000000 +0100
--- src/option.c	2011-04-15 20:49:54.000000000 +0200
***************
*** 3846,3851 ****
--- 3846,3853 ----
  # ifndef OS2	/* Always use bourne shell style redirection if we reach this */
  	    if (       fnamecmp(p, "sh") == 0
  		    || fnamecmp(p, "ksh") == 0
+ 		    || fnamecmp(p, "mksh") == 0
+ 		    || fnamecmp(p, "pdksh") == 0
  		    || fnamecmp(p, "zsh") == 0
  		    || fnamecmp(p, "zsh-beta") == 0
  		    || fnamecmp(p, "bash") == 0
***************
*** 3853,3858 ****
--- 3855,3862 ----
  		    || fnamecmp(p, "cmd") == 0
  		    || fnamecmp(p, "sh.exe") == 0
  		    || fnamecmp(p, "ksh.exe") == 0
+ 		    || fnamecmp(p, "mksh.exe") == 0
+ 		    || fnamecmp(p, "pdksh.exe") == 0
  		    || fnamecmp(p, "zsh.exe") == 0
  		    || fnamecmp(p, "zsh-beta.exe") == 0
  		    || fnamecmp(p, "bash.exe") == 0
*** ../vim-7.3.162/src/version.c	2011-04-21 14:27:21.000000000 +0200
--- src/version.c	2011-04-28 12:56:03.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     163,
  /**/

-- 
If you put 7 of the most talented OSS developers in a room for a week
and asked them to fix a bug in a spreadsheet program, in 1 week
you'd have 2 new mail readers and a text-based web browser.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
